### ISSUE TITLE
locale::facet::_S_create_c_locale name not valid

### ISSUE THREAD

**Issue by krzys-h** [colobot/colobot#445]
I'm not sure what causes that, I just compiled latest dev on Windows using MSYS2.
```
terminate called after throwing an instance of 'std::runtime_error'
  what():  locale::facet::_S_create_c_locale name not valid

This application has requested the Runtime to terminate it in an unusual way.
Please contact the application's support team for more information.
```

**Comment by krzys-h** [comment]
I've got it to work like that, but interface translations aren't working:
```
krzys_h@LaptopKrzysia /c/Users/krzys_h/Desktop/colobot-git/build
$ export LC_ALL="C"

krzys_h@LaptopKrzysia /c/Users/krzys_h/Desktop/colobot-git/build
$ ./colobot -language pl
```

**Comment by krzys-h** [comment]
This might be fixed after 87cf056158c9cb208b6fb2701118c82a2f29e660 but still needs more testing on diffrent system configurations.

**Comment by krzys-h** [comment]
Everything seems to work fine now on my old, broken configuration so I'm assuming it's fixed. I'm still getting a warning about failing to set C++ locale, through.
